released/packages/coq-elpi/coq-elpi.2.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Elpi extension language for Coq"
+description: """\
+Coq-elpi provides a Coq plugin that embeds ELPI.
+It also provides a way to embed Coq's terms into λProlog using
+the Higher-Order Abstract Syntax approach
+and a way to read terms back.  In addition to that it exports to ELPI a
+set of Coq's primitives, e.g. printing a message, accessing the
+environment of theorems and data types, defining a new constant and so on.
+For convenience it also provides a quotation and anti-quotation for Coq's
+syntax in λProlog.  E.g. `{{nat}}` is expanded to the type name of natural
+numbers, or `{{A -> B}}` to the representation of a product by unfolding
+ the `->` notation. Finally it provides a way to define new vernacular commands
+and
+new tactics."""
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: "Enrico Tassi"
+license: "LGPL-2.1-or-later"
+tags: [
+  "category:Miscellaneous/Coq Extensions"
+  "keyword:λProlog"
+  "keyword:higher order abstract syntax"
+  "logpath:elpi"
+]
+homepage: "https://github.com/LPCIC/coq-elpi"
+bug-reports: "https://github.com/LPCIC/coq-elpi/issues"
+depends: [
+  "ocaml" {>= "4.09.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.10.0"}
```
@gares from what I can tell, `CList.concat_map` that this package uses is only available in OCaml 4.10.0 and later. I'm not sure this is expected, but this is the only way I see to fix this.

Also, maybe this package is compatible with 8.19+rc1?

Reply:
Hum, CList is the coq copy of lists, I thought that would let me lower the dependency.

About 8.19, not yet, see https://github.com/coq/coq/pull/18437 and https://github.com/LPCIC/coq-elpi/pull/564